ui/pb/progressbar_test.go

```diff
@@ -101,20 +106,18 @@ func TestProgressBarRenderPaddingMaxLeft(t *testing.T) {
 		left     string
 		expected string
 	}{
-		{-1, "left", "left [--------------------------------------]"},
-		{0, "left", "left [--------------------------------------]"},
-		{15, "left_pad",
-			"left_pad        [--------------------------------------]"},
+		{-1, "left", "left   [--------------------------------------]"},
+		{0, "left", "left   [--------------------------------------]"},
 		{10, "left_truncated",
-			"left_tr... [--------------------------------------]"},
+			"left_tr...   [--------------------------------------]"},
```

Comment:
I'm wondering if we shouldn't pick a status symbol for `Running`. If we just leave it be a single space, executors with too longer names would look very strange, as you've shown in this test... To me, this seems broken, especially if you have just a single executor (so there aren't any others with different statuses to compare it against).

```
left_tr...   [--------------------------------------]  // current, seems somewhat broken
left_tr... ▶ [--------------------------------------]  // some other unicode character for running?
left_tr... » [--------------------------------------]  // some other unicode character for running?
left_trun... [--------------------------------------]  // if we reclaim the empty space when we're running, complicated to implement

// another option that might also be complicated to implement and
// introduces new state, but seems reasonably nice as UX, since it's
// going to be dynamic - every 100ms the symbol would change between:
left_tr... / [--------------------------------------]  
left_tr... - [--------------------------------------]  
left_tr... \ [--------------------------------------]  
left_tr... | [--------------------------------------]  

// or maybe there are better unicode characters we can cycle to indicate progress?
```

In any case, this is not a deal breaker and fixing it can probably come as a separate PR, just wanted to mention it here...

Reply:
Agreed, a spinning pipe would be great, and yeah, let's do that in a follow-up PR.